src/index.ts

```diff
@@ -112,16 +112,16 @@ export const checkOptions = async (opt: Options): Promise<[string[], string[]]>
 
   // Check git status if the write flag was provided and the output folder is the same as the project folder
   // Do not allow overwriting files with previous changes on them unless --ignoreGitStatus flag was provided
-  if (opt.write && path.dirname(opt.tsconfig) === opt.outputFolder) {
+  if (!opt.ignoreGitStatus && opt.write && path.dirname(opt.tsconfig) === opt.outputFolder) {
```

Comment:
It doesn't make sense to run any of the code in this block if `ignoreGitStatus` is `true`.

It will also just crash if the tsconfig file isn't in the root directory of the git project, which I didn't opt to fix in this PR.

Reply:
Yeah, I noticed this one when I tried ts-fix last.